### PR TITLE
Update queries using "region" fields

### PR DESCRIPTION
**Pull request by critzo**
This PR updates all queries that have annotations in `client.Geo.Region` or `client.Geo.Subdivision1ISOCode`, depending on the annotator used. For reference, there are three time periods of interest related to different annotation services and source data, which are currently being unified:
| **Geo Annotations** | Dates in use | **TCP Statistics Source** | Dates in use |
|:-------------------:|:------------:|:-------------------------:|:------------:|
| geo1, annotation-service | 2009-01 - 2017-08 | web100 | 2009-01 - 2019-11 |   |
| geo2, annotation-service | 2017-09 - 2020-02 | tcpinfo | 2019-11 - present | 
| geo2, uuid-annotator     | 2020-03 - present |  |  |

This PR has been tested with data back to 2018-01-01, so should be suitable to use for generating statistics for 2018 to present, while work is ongoing to re-annotate geo1 annotated data and data types still using the annotation-service.

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/stats-pipeline/53)

### CODE REVIEW

**Review by robertodauria — APPROVED** [review]
:lgtm:

> Reviewed 8 of 8 files at r1.
> *[Reviewable](https://reviewable.io/reviews/m-lab/stats-pipeline/53#-:-MVNHT1vASbU-vH484Gx:bnfp4nl)* status: :shipit: complete! 2 of 1 approvals obtained

**Review by stephen-soltesz — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/m-lab/stats-pipeline/53)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @critzo)

___
*[statistics/queries/continent_country_region_asn_histogram.sql, line 14 at r1](https://reviewable.io/reviews/m-lab/stats-pipeline/53#-MVXcR_B7mI1ufM40IaD:-MVXcR_C2SeSagYVskdp:b-vwqr7l) ([raw file](https://github.com/m-lab/stats-pipeline/blob/4aa885f854ca03db785268003876bbf920731fc1/statistics/queries/continent_country_region_asn_histogram.sql#L14)):*
> ```TSQL
>     client.Geo.ContinentCode AS continent_code,
>     client.Geo.CountryCode AS country_code,
>     CASE WHEN node._instruments IN ("tcpinfo", "web100") 
> ```

@critzo I was thinking about this PR this morning. Ideally, we want incremental improvements to the annotation-service annotations to be automatically picked up by these queries as soon as they're available.

I think a way to do that would be psuedo logic like:

```
WHEN client.Geo.Subdivision1ISOCode != "" and client.Geo.Subdivision1ISOCode is not null
    THEN CONCAT(client.Geo.CountryCode,"-",client.Geo.Subdivision1ISOCode)
ELSE
    CONCAT(client.Geo.CountryCode,"-",client.Geo.region)
```

Basically, this prefers to use the subdivision code whenever it's available and falls back to Region when it is not. This change would prevent these queries from breaking as incremental improvements are put in place for more historical data.

**Comment by critzo** [comment]
@stephen-soltesz I'll make a new PR to account for this suggestion. Thanks. There is also some recent feedback about US territories for the US specific aggregations which I'd like to include as well.
